manifests/init.pp

```diff
@@ -151,6 +157,17 @@
       key     => $certs::apache::apache_key,
       ca_cert => $certs::ca_cert,
       require => Class['certs::apache'],
+      port    => $docker_registry_port,
+    }
+  }
+
+  if $pulp_master {
+    $check_file = file('/usr/sbin/foreman-rake', '/dev/null')
+    if $check_file != '' {
+      foreman_config_entry { 'pulp_docker_registry_port':
```

Comment:
Discussion on irc leads me to think that I'm going to move the port to katello.yml instead of the settings page.